src/entity.js

```diff
@@ -695,9 +697,9 @@ Entity.setup = function(options) {
   }
 
   // Reporter for statistics
-  var reporter = function() {};
-  if (options.drain) {
-    reporter = series.AzureTableOperations.reporter(options.drain);
+  var reporter = null;
+  if (options.monitor) {
+    reporter = options.monitor;
```

Comment:
isn't the variable `monitor` better? Or just use `options.monitor` instead of creating a new variable...